### PR TITLE
Feature readme for hackathon submission

### PR DESCRIPTION
**Pull request by danielssonn**
## Purpose
Readme file for hack submission
## Screenshots
N/A
## Testing Instructions
N/A
## Checklist

- [ ] Title follows the [naming convention](../README.md#pull-requests).
- [ ] Add a description.
- [ ] Add screenshots (optional).
- [ ] Add testing instructions.
- [ ] Add at least one reviewer.
- [ ] Apply related labels (optional).
- [ ] Leave comments on changes you would like to discuss.
- [ ] Remove changes that are not related to this pull request.
- [ ] Remove debugging code.
- [ ] Test your changes locally and on staging (if applicable).
- [ ] Ensure test coverage value in config matches actual coverage (if applicable).
- [ ] Ensure the build is passing.

### CODE REVIEW

**Review by mattxyzeth — APPROVED** [review]
Looks great to me.

**Review by dobertRowneySr — APPROVED** [review]
great job, there is just a conflict to resolve

File: README.md [hunk 1]
```diff
@@ -1,15 +1,87 @@
...
 
-## Scaffolding the oracles
 
-cd /oracles,
-npm install
+
+
+
+
+# The future is multi chain, but where is your wallet?
```
Comment (karlpawlowicz) [inline README.md]:
```suggestion
# The future is multi-chain, but where is your wallet?
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,87 @@
...
+
+## Creating seamless cross chain NFT Renting experience
```
Comment (karlpawlowicz) [inline README.md]:
```suggestion
## Creating a seamless cross chain NFT renting experience
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,87 @@
...
+ * [Why we built it](#why-we-built-it)
+ * [What we built](#What-we-built)
+ * [How we built it](#How-we-built)
+
+
+
+ ### Why we built it
+
++ There are many blockchains out there, each with great assets and communities
++ The future is definitely multi chain!
```
Comment (karlpawlowicz) [inline README.md]:
```suggestion
+ The future is definitely multi-chain!   
```

File: README.md [hunk 4]
```diff
@@ -1,15 +1,87 @@
...
+ * [What we built](#What-we-built)
+ * [How we built it](#How-we-built)
+
+
+
+ ### Why we built it
+
++ There are many blockchains out there, each with great assets and communities
++ The future is definitely multi chain!   
++ However, the User eXperience interacting with multiple chains is cumbersome
```
Comment (karlpawlowicz) [inline README.md]:
```suggestion
+ However, the User eXperience of interacting with multiple chains is cumbersome
```

File: README.md [hunk 5]
```diff
@@ -1,15 +1,87 @@
...
+    + Mutiple wallets need to be created and navigated, by the user
```
Comment (karlpawlowicz) [inline README.md]:
```suggestion
    + Multiple wallets need to be created and navigated by the user
```

File: README.md [hunk 6]
```diff
@@ -1,15 +1,87 @@
...
+    + Tokens need to be swapped around, also by the user
+    + It is so easy to make a mistake, loose on the opportunity, or simply get discouraged
```
Comment (karlpawlowicz) [inline README.md]:
```suggestion
    + It is so easy to make a mistake, lose out on an opportunity, or simply get discouraged
```

File: README.md [hunk 7]
```diff
@@ -1,15 +1,87 @@
...
+
+ ### What we built
+ + Cross chain NFT renting experience
```
Comment (karlpawlowicz) [inline README.md]:
```suggestion
 + Cross-chain NFT renting experience
```

File: README.md [hunk 8]
```diff
@@ -1,15 +1,87 @@
...
+ + With simple and straightforward UX
+ + No matter what chain and which wallet or tokens you have 
+ + You can list and rent NFTs on multiple chains, without the multi chain hassle
```
Comment (karlpawlowicz) [inline README.md:57]:
```suggestion
 + You can list and rent NFTs on multiple chains without the multi-chain hassle
```

File: README.md [hunk 9]
```diff
@@ -1,15 +1,87 @@
...
+
+ 
+ ### How we built it
+ + We have created NFT renting contracts for lenders and renters
+ + We have deployed the contracts on Polygon and Moonbeam
+    - [Mumbai Polygonscan](https://mumbai.polygonscan.com/address/0x8c787c95e9f1bbc6153336571b7ab58cd57ad98c) 
+    - [Moonbase Alpha](https://moonbase.moonscan.io/token/0xec19ebb094269b2782fbab3b5ce6e3cb4dea86a2)
+    
+ + Mixed in some Axelar magic to make the contracts **cross chain BFF!**
```
Comment (karlpawlowicz) [inline README.md]:
```suggestion
 + Mixed in some Axelar magic to make the contracts **cross-chain BFF!**
```

**Review by karlpawlowicz — APPROVED** [review]
Looks great! I left a few small suggestions

**Pull request merged**